pkg/apis/policy/v1alpha1/egress.go

```diff
@@ -34,6 +34,10 @@ type EgressSpec struct {
 	// +optional
 	Hosts []string `json:"hosts,omitempty"`
 
+	// IPAddresses defines the list of external IP addresses the Egress policy is applicable to
```

Comment:
suggesting?
```suggestion
	// IPAddresses defines the list of external destination IP addresses the Egress policy is applicable to
```

Reply:
Doesn't external imply destination?

Reply:
you can totally skip, no big deal
Reading the line in isolation, one could think the egress policy is applicable or filters source instead